src/fides/api/ops/schemas/connection_configuration/connection_secrets_dynamodb.py

```diff
@@ -0,0 +1,24 @@
+from typing import List, Optional
+
+from fides.api.ops.schemas.base_class import NoValidationSchema
+from fides.api.ops.schemas.connection_configuration.connection_secrets import (
+    ConnectionConfigSecretsSchema,
+)
+
+
+class DynamoDBSchema(ConnectionConfigSecretsSchema):
+    """Schema to validate the secrets needed to connect to an Amazon DynamoDB cluster"""
+
+    region_name: Optional[str]
```

Comment:
Would it be more accurate for these to be required string type instead of optional given the `_required_components` definition?